vcd/datasource_vcd_vdc_group.go

```diff
@@ -0,0 +1,199 @@
+package vcd
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+)
+
+func datasourceVdcGroup() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: datasourceVcdVdcGroupRead,
+
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"name": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				Computed:     true,
+				ExactlyOneOf: []string{"name", "id"},
+				Description:  "Name of VDC group",
+			},
+			"id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				Computed:     true,
+				ExactlyOneOf: []string{"name", "id"},
+				Description:  "VDC group ID",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "VDC group description",
+			},
+			"dfw_enabled": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Distributed firewall status",
+			},
+			"default_policy_status": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: "Default Policy Status",
+			},
+			"error_message": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "More detailed error message when VDC group has error status",
+			},
+			"local_egress": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: "Status whether local egress is enabled for a universal router belonging to a universal vDC group.",
```

Comment:
There are twenty incorrect `vDC` capitalizations in this PR, which should be replace to `VDC`.

Reply:
It comes from VCD documentation, but changed.